### PR TITLE
[RHPAM-3181] Update the openshift.PING to kubernetes.PING

### PR DESCRIPTION
**Pull request by desmax74**
@spolti @jakubschwan 
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: templates/docs/rhpam710-authoring-ha.adoc [hunk 1]
```diff
@@ -342,9 +342,7 @@ tcpSocket on port 3306
 |`HTTPS_KEYSTORE` | The name of the keystore file within the secret for Business Central. | `${BUSINESS_CENTRAL_HTTPS_KEYSTORE}`
 |`HTTPS_NAME` | The name associated with the server certificate for Business Central. | `${BUSINESS_CENTRAL_HTTPS_NAME}`
 |`HTTPS_PASSWORD` | The password for the keystore and certificate for Business Central. | `${BUSINESS_CENTRAL_HTTPS_PASSWORD}`
-|`JGROUPS_PING_PROTOCOL` | -- | openshift.DNS_PING
-|`OPENSHIFT_DNS_PING_SERVICE_NAME` | -- | `${APPLICATION_NAME}-rhpamcentr-ping`
-|`OPENSHIFT_DNS_PING_SERVICE_PORT` | -- | 8888
+|`JGROUPS_PING_PROTOCOL` | -- | kubernetes.KUBE_PING
```
Comment (spolti) [inline templates/docs/rhpam710-authoring-ha.adoc:345]: no need to update adocs, it is auto generated before releases.

File: templates/docs/rhpam710-authoring-ha.adoc [hunk 2]
```diff
@@ -516,55 +514,17 @@ kieserver-app-secret
...
-    name: eap-app-ping
-    annotations:
-        service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
-        description: "The JGroups ping port for clustering."
-----
 
 For `KUBE_PING` to work, the following steps must be taken:
 
-. The `OPENSHIFT_KUBE_PING_NAMESPACE` environment variable must be set (see table above).
+. The `KUBERNETES_KUBE_PING_NAMESPACE` environment variable must be set (see table above).
```
Comment (spolti) [inline templates/docs/rhpam710-authoring-ha.adoc:527]: KUBERNETES_NAMESPACE

plus make sure all templates and operator sets this env.

**Review by spolti — CHANGES_REQUESTED** [review]

**Comment by desmax74** [comment]
@spolti

**Review by msmagnanijr — DISMISSED** [review]

File: templates/docs/rhpam710-authoring-ha.adoc [hunk 3]
```diff
@@ -516,57 +516,19 @@ kieserver-app-secret
 
 Clustering in OpenShift EAP is achieved through one of two discovery mechanisms:
 Kubernetes or DNS. This is done by configuring the JGroups protocol stack in
-standalone-openshift.xml with either the `<openshift.KUBE_PING/>` or `<openshift.DNS_PING/>`
```
Comment (spolti) [inline templates/docs/rhpam710-authoring-ha.adoc:519]: this needs to be c hanged on the jboss-kie-modules repo on the temp[alte adoc.
templates' adoc files are generated, no need to update it manually.

**Review by spolti — CHANGES_REQUESTED** [review]

File: templates/docs/rhpam710-authoring-ha.adoc (hunk 1 above)
Comment (desmax74) [reply templates/docs/rhpam710-authoring-ha.adoc]: ok

File: templates/docs/rhpam710-authoring-ha.adoc (hunk 3 above)
Comment (desmax74) [reply templates/docs/rhpam710-authoring-ha.adoc]: Ok

**Comment by desmax74** [comment]
Replaced by https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/504

**Pull request closed without merging**